nym-vpn-core/crates/nym-vpnc/src/main.rs

```diff
@@ -141,31 +159,113 @@ async fn connect(client_type: ClientType, connect_args: &cli::ConnectArgs) -> Re
     });
 
     let response = client.vpn_connect(request).await?.into_inner();
-    println!("{:#?}", response);
+
+    if opts.verbose {
+        println!("{:#?}", response);
```

Comment:
should all of those responses be printed with `println` or rather `tracing::...`?

Reply:
Currently there is no logger setup at all. Honestly I'm not sure. Usually you don't have a logger for interactive CLI apps? Or should we?